### ISSUE TITLE
was Avocado an early name for Avo?

### ISSUE THREAD

**Issue by brandondrew** [avo-hq/docs#43]
The Avo docs have a tagline of "The documentation for Avocado".  Is this intentional?

Also, the URL is currently https://goofy-docs-69c89b.netlify.app/, which displays a page with 

```
Not Found - Request ID: 01G6VKX8CHCV3X0N2YPH8B097N
```

I'm guessing that [the current documentation](https://docs.avohq.io/2.0/) is meant to be used instead?

**Comment by adrianthedev** [comment]
Yeah. It was named project Avocado but I didn't find any good domain or social media handles.

Thanks for the nudge.

The logo on the docs page is the old one too, but I'm keeping that there to nudge me to change the documentation site to something more modern.
